OrbitGrpcProtos/capture.proto

```diff
@@ -177,6 +177,50 @@ message AddressInfo {
   }
 }
 
+message GpuQueueSubmission {
+  GpuQueueSubmissionMetaInfo meta_info = 1;
+  repeated GpuSubmitInfo submit_infos = 2;
+  repeated GpuDebugMarker completed_markers = 3;
+  uint32 num_begin_markers = 4;
+}
+
+message GpuQueueSubmissionMetaInfo {
+  int32 thread_id = 1;
+  uint64 pre_submission_cpu_timestamp = 2;
+  uint64 post_submission_cpu_timestamp = 3;
+}
+
+message GpuSubmitInfo {
+  repeated GpuCommandBuffer command_buffers = 1;
+}
+
+message GpuCommandBuffer {
+  uint64 begin_gpu_timestamp_ns = 1;
+  uint64 end_gpu_timestamp_ns = 2;
+}
+
+message GpuDebugMarkerBeginInfo {
+  GpuQueueSubmissionMetaInfo meta_info = 1;
+  uint64 gpu_timestamp_ns = 2;
+}
+
+message GpuDebugMarker {
+  oneof begin_marker_option {
+    GpuDebugMarkerBeginInfo begin_marker = 1;
+  };
+  uint64 end_gpu_timestamp_ns = 2;
+  uint64 text_key = 3;
+  uint64 depth = 4;
```

Comment:
This should probably be `int32`, similarly to `FunctionCall`, unless you have a specific reason why you need it unsigned (and 64 bits).

Reply:
Done.